### ISSUE TITLE
Current status?

### ISSUE THREAD

**Issue by mkuron** [struegamer/proteus-api#1]
Did you ever get this implemented? If so, it would be great if you could publish your code to the repo.
I'm looking for a simple way to push IP and name changes to Proteus from a script, so if you already have written Python wrappers around the SOAP API, I'd love to use that.

**Comment by struegamer** [comment]
Hi

I am just working on this right now

I'll let you know when it's pushed

Regards

\sh

**Comment by struegamer** [comment]
@mkuron can you test something for me?

**Comment by mkuron** [comment]
Not yet, I first need to get an API account from my IPAM admin.
From the Proteus documentation I had feared that there would be no way to create a restricted API account (I am just a user on a shared Proteus box), but as I just found out the current version allows exactly this, so I should be able to get my admin to give me an API account for my domains and subnets.

Thanks for publishing your code! Looks like a nice bit of work (and more to come I'm sure). I'll let you know once I'm ready for some testing.
